File: recipes_3e/apache/web_connect/web_connect.rb
#!/usr/bin/ruby

require "Cookbook"

# Print header, blank line, and initial part of page

puts "Content-Type: text/html"
puts ""

puts "
<html>
<head><title>Ruby Web Connect Page</title></head>
<body>
"

# Connect to and disconnect from database
dbh = Cookbook.connect
puts "<p>Connected</p>"
dbh.disconnect
puts "<p>Disconnected</p>"

# Print page trailer
puts "
</body>
</html>
"

File: recipes_3e/apache/web_connect/web_connect.pl
#!/usr/bin/perl

use strict;
use warnings;
use Cookbook;

# Print header, blank line, and initial part of page
print <<EOF;
Content-Type: text/html

<html>
<head><title>Perl Web Connect Page</title></head>
<body>
EOF
# Connect to and disconnect from database
my $dbh = Cookbook::connect ();
print "<p>Connected</p>\n";
$dbh->disconnect ();
print "<p>Disconnected</p>\n";
# Print page trailer
print <<EOF;
</body>
</html>
EOF
